### PR TITLE
Feature/asb 1176 relevant reps db schema

### PR DESCRIPTION
**Pull request by DavidBeale**
## Describe your changes

Initial schema for Relevant Reps

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/prisma/schema.prisma [hunk 1]
```diff
@@ -318,7 +320,9 @@ model Document {
   createdAt            DateTime @default(now())
   fileSize             Int?     @default(0)
   fileType             String?
+  representationId     Int?
 
+  representations  Representation?   @relation(fields: [representationId], references: [id])
```
Comment (Philip-Philippou1) [inline apps/api/prisma/schema.prisma:325]: Can we confirm this link between user and representation.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
-services: 
+services:
   rabbitmq:
     image: rabbitmq:3.10-management
     ports:
       - 5672:5672
       - 15672:15672
     volumes:
       - ./rabbitmq_config:/etc/rabbitmq
+
```
Comment (Philip-Philippou1) [inline docker-compose.yml:11]: Couldn't get this to work on an M1 mac, we can go through it though.

File: apps/api/prisma/schema.prisma (hunk 1 above)
Comment (DavidBeale) [reply apps/api/prisma/schema.prisma:325]: This is the representation to document one-to-many

File: docker-compose.yml (hunk 2 above)
Comment (DavidBeale) [reply docker-compose.yml:11]: You may need to update and set some settings https://github.com/microsoft/mssql-docker/issues/668#issuecomment-1436802153

File: apps/api/prisma/schema.prisma (hunk 1 above)
Comment (Philip-Philippou1) [reply apps/api/prisma/schema.prisma:325]: There is a link to User, will show you.

**Review by Philip-Philippou1 — DISMISSED** [review]

File: apps/api/prisma/schema.prisma [hunk 3]
```diff
@@ -343,12 +349,14 @@ model DocumentVersion {
   mime                   String?
   horizonDataID          String?
   fileMD5                String?
+  documentId             String?
```
Comment (mayya-bond-kc) [inline apps/api/prisma/schema.prisma:352]: @LetsMakeItTechnical Would be good to come to an agreement regarding what we mean by `id`, `documentId`, `horizonDataId` with @DavidBeale @Philip-Philippou1

**Review by mayya-bond-kc — APPROVED** [review]
LGTM, lets just check we are in agreement regarding doc references/ids

File: apps/api/prisma/schema.prisma (hunk 3 above)
Comment (LetsMakeItTechnical) [reply apps/api/prisma/schema.prisma:352]: documentId and version will need to be a primary key

Comment (DavidBeale) [reply apps/api/prisma/schema.prisma:352]: Seems like this has had one rebase too many - so I will clean it up and push again

**Review by mayya-bond-kc — APPROVED** [review]

**Pull request merged**